Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
why not run `make test`? A binary is being released, seems very important to test it

the only test that fails is a catch for seg faults, it's because of how ruby's `system` works. i'm trying to figure out how to avoid running it right now